packages/protocol/contracts/tokenvault/ERC20Vault.sol

```diff
@@ -437,4 +442,11 @@ contract ERC20Vault is BaseVault {
             ctokenDecimal: ctoken.decimals
         });
     }
+
+    function _checkRateLimit(address _ctoken, uint256 _amount) private {
+        address rateLimiter = resolve(LibStrings.B_RATE_LIMITER, true);
+        if (rateLimiter != address(0)) {
+            IRateLimiter(rateLimiter).consumeAmount(_ctoken, _amount);
+        }
+    }
```

Comment:
Function name `_checkRateLimit` a little bit ambigous to me, since it callse consumeAmount, which i guess eventually will deduct some balance from something, so not only 'checking'.

Reply:
renamed